src/lib/expansion/expansion-panel.ts

```diff
@@ -127,12 +147,31 @@ export class MatExpansionPanel extends _MatExpansionPanelMixinBase
     return this.expanded ? 'expanded' : 'collapsed';
   }
 
+  ngAfterViewInit() {
```

Comment:
We have had a number of issues with content changed after checked for these situations, are we going to run into the same issue here?

Reply:
Probably not, those exceptions are usually if data bindings change. This is the same way we're doing it for tabs.

Reply:
For tabs we actually are having to change it because of these issues #7266 

We are creating a custom template portal and having the template portal itself be responsible for attachment because of issues with `ChangeAfterContentChecked`.  I just want to be sure we don't accidentally create an issue further down the road.